queue/src/fiskaltrust.Middleware.Queue/SignProcessor.cs

```diff
@@ -71,7 +71,14 @@ public async Task<ReceiptResponse> ProcessAsync(ReceiptRequest request)
 
                 var queue = await _configurationRepository.GetQueueAsync(_middlewareConfiguration.QueueId).ConfigureAwait(false);
 
-                return await InternalSign(queue, request).ConfigureAwait(false);
+                var response = await InternalSign(queue, request).ConfigureAwait(false);
+
+#if NET6_0_OR_GREATER
+                System.Diagnostics.Activity.Current?.AddTag("queue.ReceiptResponse.ftState", $"0x{response.ftState:X}");
```

Comment:
response can be null in case of using the flag too lookup a receipt and not getting back anything :/